lib/rubocop/cop/sequel/not_valid_constraint.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Sequel
+      # Constraint created without not_valid option
+      class NotValidConstraint < Base
+        MSG = 'Prefer creating new constraint with the `not_valid: true` option.'
+        RESTRICT_ON_SEND = %i[add_foreign_key add_constraint].freeze
```

Comment:
Probably, I should add `constraint` also as it is an "alias" to `add_constraint`:
https://github.com/jeremyevans/sequel/blob/master/lib/sequel/extensions/constraint_validations.rb#L231-L234

Reply:
Added here: [715f3d4](https://github.com/rubocop/rubocop-sequel/pull/36/commits/715f3d4eb26e25b37d8b8f570b92a878e79a3a8f)